tox.ini

```
@@ -53,27 +52,22 @@ deps =
     -rlint-requirements.txt
 extras = []
 skip_install = true
-tags =
-    lint
 usedevelop = false
 
 [testenv:format]
 commands =
     yapf -i -r molecule/ test/
 deps = yapf>=0.25.0,<2
-extras =
+extras = []
```

It's not needed. Keep it empty/reset.
```suggestion
extras
```

---

I aligned on the lint section to make the `tox.ini` file consistent.